agent/app/agent_unix.go

```diff
@@ -48,7 +48,7 @@ const initPID = 1
 var awsVPCCNIPlugins = []string{ecscni.ECSENIPluginName,
 	ecscni.ECSBridgePluginName,
 	ecscni.ECSIPAMPluginName,
-	ecscni.ECSAppMeshPluginName,
+	//ecscni.ECSAppMeshPluginName,
```

Comment:
why are we disabling this plugin?

Reply:
Appmesh is not yet supported with how the CNI plugins were installed. This will be added to future work for the project. It was pushed out of scope for now. 
Bottlerocket was referred to while installing CNI plugins: https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/ecs-agent/0004-bottlerocket-remove-unsupported-CNI-plugins.patch